.github/workflows/outdated-integrations.yml

```diff
@@ -0,0 +1,39 @@
+name: Outdated Integrations
+on:
+  workflow_dispatch: # manually
+  push:
+    branches:
+      - quinna.halim/update-outdated-integrations-workflow # TODO: remove, for testing
+
+concurrency:
+  # Automatically cancel previous runs if a new one is triggered to conserve resources.
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Find new major versions for the contrib package dependencies
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: write
+      pull-requests: write
+    env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v4
+      
+      - run: go run latest_major_version.go > latest_majors.txt
+
+      - run: git diff
+
+      - name: Create Pull Request
+        id: pr
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: "upgrade-latest-major-version"
+          commit-message: "Update latest majors file"
+          base: main
+          title: "chore: update latest majors"
+          labels: changelog/no-changelog
```

Comment:
Could we just add a `body` here for something simple like

`body: "Updating latest majors with new versions"`

Just so the body of the PR isn't a link back to the peter evans workflow